test/models/ocelot/LabelSpec.scala

```diff
@@ -45,4 +45,38 @@ class LabelSpec extends BaseSpec with ProcessJson {
 
   }
 
+  "LabelCache" must {
```

Comment:
You could add a test for the interesting case of a label with an undefined value. If I'm right value will return Some().

Reply:
Some("") yes, and non-existent None, will add test